source/admin_manual/list_of_events.rst

```diff
@@ -1229,6 +1270,27 @@ deleted.
 | reason               | Reason string why cache was found to be corrupted.    |
 +----------------------+-------------------------------------------------------+
 
+.. _event_mail_cache_lookup:
+
+mail_cache_lookup
+^^^^^^^^^^^^^^^^^
```

Comment:
Should actually rename this to mail_cache_lookup_finished for consistency.

Reply:
changed